apps/Gruntfile.js

```diff
@@ -21,6 +21,7 @@ module.exports = function (grunt) {
     // get compiled.
     fs.writeFileSync(
       'test/entry-tests.js',
+      "require('babel-polyfill');\n" +
```

Comment:
`entry-tests.js` is an auto generated file. These lines right here are what generate that file. Nobody should be adding anything to it.